ArduPlane/Plane.h

```diff
@@ -517,6 +517,28 @@ class Plane : public AP_Vehicle {
         // throttle  commanded from external controller in percent
         float forced_throttle;
         uint32_t last_forced_throttle_ms;
+
+#if OFFBOARD_GUIDED == ENABLED
+        // airspeed adjustments
```

Comment:
These should probably be grouped into a struct, as these would easily seem to overlap with other internal members.

Reply:
... they are already in a struct called 'guided_state'... maybe u missed that?